recipes/_master_package.rb

```diff
@@ -46,13 +46,32 @@
 when 'rhel'
   include_recipe 'yum::default'
 
+  ruby_block "Security off" do
```

Comment:
It is, without security off, you can't install and configure jenkins from the cli. In a wrapper cookbook, a user can turn security on, which should probably be in the documentation with an example, but to be able to run this cookbook, it needs to be off.
Otherwise, you need to configure a public and private key pair for the main user after jenkins install.